scripts/release/group_test.py

```diff
@@ -52,6 +52,11 @@ def run_pytest_jobs(pytest_jobs: dict, my_env: dict):
     workspace = args.workspace_name
     coverage_report = args.coverage_report
 
+    # change current dir to test dir
+    cwd = os.getcwd()
+    os.chdir(tests_dir)
+    logger.print(f"pushd: {os.getcwd()}")
```

Comment:
Could we do this as part of a context manager?

Reply:
Added